resources/styles/global/forms.less

```diff
@@ -7,4 +7,33 @@ textarea {
   }
 }
 
+
+.form-control:focus, .form-control.focus {
```

Comment:
Should these be nested inside `form-control-wrapper` and then deduplicated with the other `.form-control` ?